editor/create_dialog.cpp

```diff
@@ -243,7 +244,13 @@ void CreateDialog::_add_type(const String &p_type, const TypeCategory p_type_cat
 	} else {
 		if (ScriptServer::is_global_class(p_type)) {
 			Ref<Script> scr = EditorNode::get_editor_data().script_class_load_script(p_type);
-			ERR_FAIL_COND(scr.is_null());
+			if (scr.is_null()) {
+				// Remove non-existing global class from class cache.
+				ScriptServer::remove_global_class(p_type);
+				ScriptServer::save_global_classes();
+				print_line("Removed class " + p_type + " from global classes.");
```

Comment:
```suggestion
				print_verbose("Removed class " + p_type + " from global classes.");
```
Or no printing here at all IMO

Reply:
In my opinion, the best solution would be totally silent and just to remove the class from cache file. User doesn't care. But suppressing those error messages wouldn't be easy. So I added that print to show the user that _something_ was done after those errors. I fear that `print_verbose` would mean that next to nobody would see the message. Then removing the whole message would be as good.

Reply:
I don't agree, the message matters for verbose printing to debug things, but removing the printing all together is fine, but it has some value, but printing it always isn't good